.travis.yml

```diff
@@ -6,12 +6,12 @@ sudo: false
 
 python:
   - 3.5
-  - 2.7
```

Comment:
May need to add this back in, depends what Divio state for this as the CMS is still 2.7 compatible.